materializer/nestedfieldselection/api.graphql

```diff
@@ -0,0 +1,84 @@
+# See preamble in README.md
+
+# Original graph G
+type Document {
+   docId: String
+   content(id: String!): Content
+     @materializer(query: "content", arguments:[{name: "id" argument:"id"}])
```

Comment:
remove `@materializer`

Reply:
again, how to get the sample query to run?